server/package.json

```diff
@@ -19,23 +19,31 @@
     "cors": "^2.8.5",
     "express": "^4.17.1",
     "helmet": "^3.22.0",
-    "mongoose": "^5.9.18"
+    "mongoose": "^5.9.18",
+    "puppeteer": "^3.3.0",
+    "handlebars": "^4.7.6"
   },
   "devDependencies": {
     "@babel/cli": "^7.10.1",
     "@babel/core": "^7.10.2",
     "@babel/node": "^7.10.1",
     "@babel/preset-env": "^7.10.2",
     "babel-jest": "^26.0.1",
+    "compression": "^1.7.4",
+    "convert-html-to-pdf": "^1.0.1",
+    "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "eslint": "^7.2.0",
     "eslint-config-airbnb-base": "^14.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jest": "^23.13.2",
     "eslint-plugin-prettier": "^3.1.3",
+    "express": "^4.17.1",
+    "helmet": "^3.22.0",
     "jest": "^26.0.1",
-    "nodemon": "^2.0.4",
```

Comment:
`nodemon` needs to stay - we are using this for dev

Reply:
,,|,, done